code-challenges.js

```diff
@@ -30,18 +56,53 @@ var myNumbers2 = [8, -7, 22, 9, 13]
 var myString1 = "bravo"
 var myString2 = "charlie"
 
-
+// create function that takes in a single input
+// return input with .reverse() mutator
+// log the function invocation twice, each with different myString variables provided
+
+// const reverseStringMachine = (string) => {
+//     return string.reverse()
+// }
+// console.log(reverseStringMachine(myString1));
+// console.log(reverseStringMachine(myString2));
+
+// the code above gave me an error
+// after reading through the syllabus I realized the .reverse() method only works on arrays.. duh!
+// the input, a string, first needs to be .split("") which will convert it to an array
+// the array can then be reversed with the .reverse() mutator
+// then .join("") to change the reversed array back into a reversed string
+// it worked!!! great success!
```

Comment:
This is exactly the kind of psuedo code and error logging we want to see! Great work!